sonar-scanner-engine/src/test/resources/org/sonar/scanner/issue/ignore/scanner/IssueExclusionsRegexpScannerTest/file-with-single-regexp.txt

```diff
@@ -30,4 +27,5 @@ public class LineRange {
     return lines;
   }
 
-}
\ No newline at end of file
+}
+// @SONAR-IGNORE-ALL
```

Comment:
Don't you mix file name? Isn't this one supposed to be 'file-with-single-regexp-last-line.txt' ?